snapshots/who/2025-01-09/vaccination_schedules.xlsx.dvc

```diff
@@ -0,0 +1,30 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: WHO Immunization Data
+    description: |
+
+    date_published: "2024-03-01"
+    title_snapshot: WHO Immunization Data - Vaccination Schedules
+
+    # Citation
+    producer: World Health Organization
+    citation_full: |-
+      Vaccine Schedule - WHO/UNICEF Joint Reporting Form on Immunization (2024), World Health Organization and  UNICEF.
+    attribution_short: WHO
+    # Files
+    url_main: https://immunizationdata.who.int/global?topic=Vaccine-introduction&location=
+    url_download: https://srhdpeuwpubsa-geecgzbpd5h0fueu.z01.azurefd.net/whdh/WIISE/export/vaccine-introduction-data.xlsx
+    date_accessed: 2025-01-09
+
+    # License
+    license:
+      name: © WHO
```

Comment:
According to the license site, I think that the license should be "[CC BY-NC-SA 3.0 IGO](https://creativecommons.org/licenses/by-nc-sa/3.0/igo/)", right?

(first bullet point in the image)

<img width="1019" alt="image" src="https://github.com/user-attachments/assets/eac2a5f5-da4d-4137-8dec-c939ad516b3b" />